src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -292,7 +292,12 @@ public static Integer parseManpowerToAllocate(String manpowerToAllocate) throws
         }
 
         String trimmed = manpowerToAllocate.trim();
-        Integer manpowerToAdd = Integer.valueOf(trimmed);
+        Integer manpowerToAdd;
+        try {
+            manpowerToAdd = Integer.valueOf(trimmed);
+        } catch (NumberFormatException e) {
+            throw new ParseException("Number of employees to allocate must be a positive integer!");
+        }
         if (manpowerToAdd <= 0) {
```

Comment:
Can possibly move it to the block above

```
if (manpowerToAdd <= 0) {
 throw new NumberFormatException();
}
```
catch block can remain unchanged, as it will catch the negative interger above and throw a new ParseException

Reply:
Nice